### ISSUE TITLE
RI5CY: User Manual v4.2: p.clip* instructions imply ls2/rs2 unsigned rs1 signed for all

### ISSUE THREAD

**Issue by alfredoh1234** [pulp-platform/riscv#213]
RI5CY: User Manual v4.2 does not clarify which arguments are signed and which are unsigned. 

From pseudo code seems like ls2 and rs2 is always unsigned and rs1 is always signed. 

Clarification or annotation may be needed. Please clarify.

**Comment by davideschiavone** [comment]
Yes, p.clint are always signed. If not specified numbers are always signed.

**Comment by alfredoh1234** [comment]
Can there be a sentence added to the document saying that? For example between the header of section 14 and the first subsection 14.1 ?
Something like: "All numbers are signed, unless specifically indicated."
